autokeras/hypermodel/hyper_block.py

```diff
@@ -156,29 +156,46 @@ def build(self, hp, inputs=None):
 
 class ConvBlock(HyperBlock):
 
+    def __init__(self, separable=None, **kwargs):
```

Comment:
I am in general confused as to why some arguments are present in the constructor and as hps, while others are only hps.

We should follow a consistent convention here.

How about: all arguments are listed in the constructor, defaulting to None, and if they're None they get selected via a hp?

Reply:
Good idea. Since it requires massive changes to all the hyperblocks, it would be a separate pull request.